assets/models/system/bert-base-cased/spec.yaml

```diff
@@ -16,6 +16,7 @@ tags:
   license: apache-2.0
   model_specific_defaults : {'apply_deepspeed': 'true', 'apply_lora': 'true', 'apply_ort': 'true'}
   SharedComputeCapacityEnabled: ''
+  mlflow_flavor: transformers
```

Comment:
why is this required? For filteration?

Reply:
Also this will provide a way from SDK to know the flavor, currently there is no way from sdk to know flavor, exception downloading model and check for artifacts

Reply:
> Also this will provide a way from SDK to know the flavor, currently there is no way from sdk to know flavor, exception downloading model and check for artifacts

Also this will provide a way from SDK to know the flavor, currently there is no way from sdk to know flavor, exception downloading model and check for artifacts